packages/features/ee/organizations/context/provider.ts

```diff
@@ -0,0 +1,63 @@
+import { createContext, useContext, createElement } from "react";
+
+/**
```

Comment:
Used FeatureFlag provider as template, basically defines the Organization Branding Provider the same way.